migrations/seed-test-data/20190727015604-populate-test-data-in-organization-table.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+  * We receive the dbmigrate dependency from dbmigrate initially.
+  * This enables us to not have to rely on NODE_PATH.
+  */
+exports.setup = function(options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = function(db) {
+  return db.insert('organization', 
```

Comment:
thinking about this.... Is there a more succinct way to do a lot of data importing in these migrations?

Reply:
Not sure how more succinct this could be. All steps to exactly reproduce the database have to be explicitly defined.

Reply:
I think it's just that this is just a huge PR, and it's a bit unclear how it all comes together.
In terms of succinctness, I think that inserting all the information line by line could be made much more readable.